front/app/containers/Authentication/steps/AuthProviders/index.tsx

```diff
@@ -101,11 +113,22 @@ const AuthProviders = memo<Props>(
           />
         )}
 
+        {claveUnicaLoginEnabled && (
+          <>
+            {franceconnectLoginEnabled && <Or />}
+            <StyledExpandedAuthProviderButton
+              flow={flow}
+              onSelectAuthProvider={onSelectAuthProvider}
+            />
+          </>
+        )}
+
         {(isPasswordSigninOrSignupAllowed ||
```

Comment:
Be tempted to refactor this boolean logic out into a named const. Might make it easier to understand the intention at a glance